src/parameterizations/lateral/MOM_MEKE.F90

```diff
@@ -1560,49 +1562,52 @@ subroutine ML_MEKE_init(diag, G, US, Time, param_file, dbcomms_CS, CS)
   CS%id_mke = register_diag_field('ocean_model', 'MEKE_MKE', diag%axesT1, Time, &
      'Surface mean (resolved) kinetic energy used in MEKE', 'm2 s-2', conversion=US%L_T_to_m_s**2)
   CS%id_slope_z= register_diag_field('ocean_model', 'MEKE_slope_z', diag%axesT1, Time, &
-     'Vertically averaged isopyncal slope magnitude used in MEKE', 'm2 s-2', conversion=US%L_T_to_m_s**2)
+     'Vertically averaged isopyncal slope magnitude used in MEKE', 'nondim', conversion=US%Z_to_L)
```

Comment:
I'm not quit clear why there should be a conversion factor here since it is a non-dimensional quantity.

Reply:
In MOM6, we are using different dimensional scaling for horizontal distances (`[L ~> m]`) from vertical distances (`[Z ~> m]`).  As a result, the rescaled units of a slope are not simply nondimensional, but rather (`[Z L-1 ~> nondim]`), hence the conversion factor in these slope diagnostics.

   An analogy to help illustrate this might be in the units we use to characterize a hiking trail, where the lateral distance covered is typically in miles (or km), but the elevation gain is in feet (or m), so the steepness of a trail might be given in feet per mile (or m per km) or as a percentage grade (like 5% grade), and not as a simple unitless steepness, like 0.05.

Reply:
Thanks @Hallberg-NOAA I understand and agree then with the conversion factor